packages/ui/__tests__/Tooltip.test.tsx

```diff
@@ -1,64 +1,107 @@
+/* eslint-disable @typescript-eslint/require-await */
 import React from 'react'
+import { act } from 'react-dom/test-utils'
 import { mount } from 'enzyme'
-import RCTooltip from 'rc-tooltip'
 
-import { Tooltip, TooltipProps } from '../src/Tooltip'
-
-import styles from '../src/Tooltip.module.scss'
+import { Tooltip } from '../src/Tooltip'
 
+/**
+ * There's a weird issue with react-popper that results in missing act() wrapper error.
+ * This is why all mounts are wrapped in act().
+ *
+ * https://github.com/popperjs/react-popper/issues/368
+ */
 describe('Tooltip', () => {
-  test('renders children without Tooltip if "overlay" is undefined', () => {
-    const Child = () => <div />
+  test('Renders correctly if "content" property is defined.', async () => {
+    await act(async () => {
+      const wrapper = mount(
+        <Tooltip id="tooltip-test" content="Tooltip content">
+          {(ref) => (
+            <button ref={ref} data-test="tooltip-reference">
+              Hover me
+            </button>
+          )}
+        </Tooltip>,
+      )
+
+      const tooltipOverlay = wrapper.find('[data-test="tooltip-overlay"]')
+
+      expect(wrapper.find('[data-test="tooltip-reference"]').exists()).toBeTruthy()
```

Comment:
We have a helper `findDataTest` to make finding `data-test` simpler.
Instead of writing `wrapper.find('[data-test="tooltip-reference"]'` you just write `wrapper.findDataTest('tooltip-reference')`.

Reply:
Yes, back then I was having issues with `mountAndCheckA11Y()`, but @aigoncharov managed to fix it. I'll update this soon, thanks :)

Reply:
Cool :+1: